updates/version.yaml

```diff
@@ -1 +1,3 @@
 1.0.1: First version of Revisions
+1.1.0: Add Russian and German Transltions
```

Comment:
Can you remove these versions? @Samuell1 will add new versions himself I think

Reply:
Done